test/evergreen.yml

```diff
@@ -3582,6 +3608,7 @@ buildvariants:
     - name: data-validation-stress-test-checkpoint-no-timestamp
     - name: unittest-test
     - name: tiered-storage-extensions-test
+    - name: tiered-test-small
```

Comment:
I don't think this line is necessary, as the task selector above (line 3578) should have covered this new task, since it has `pull_request` tagged.

Reply:
Btw, is `tiered-test-small` actually a subset of `tiered-storage-extensions-test` in terms of testing scope?

Reply:
@lukech Yes it is just a subset!